x/cosmwasmpool/module/module.go

```diff
@@ -30,7 +30,9 @@ var (
 	_ module.AppModuleBasic = AppModuleBasic{}
 )
 
-type AppModuleBasic struct{}
+type AppModuleBasic struct {
+	cdc codec.Codec
```

Comment:
Where is this being set?

Reply:
Appreciate you catching this, set them here https://github.com/osmosis-labs/osmosis/pull/5696/commits/1d4505c2b1501111efc3019cb870b7d19e316322